testcases/android/cts.yaml

```diff
@@ -0,0 +1,23 @@
+{% extends device_type %}
+
+{% set cts_test_params = cts_test_params | default("cts") %}
+
+{% block job_specific_metadata %}
```

Comment:
You should be able to say:
{% block metadata %}
{{ super() }}
cts-url: "{{TEST_CTS_URL}}/android-cts.zip"
...
{% endblock metadata %}

Reply:
job_specific_metadata will make it more clear it's only for this job.